### ISSUE TITLE
Issues with the coordinates of area highlighter

### ISSUE THREAD

**Issue by elisabetta-riskapp** [agentcooper/react-pdf-highlighter#67]
Hello Artem,
I am having an issue with the area highlight, as you can see in the gif below. I am using the default `AreaHighlight` component, while I have slightly modified the `Highlight` and the `Tip` components (as you can see), but it's mostly aesthetic changes, I didn't change any logic behind those components. 
It happens randomly: sometimes it works and sometimes it doesn't, and I manage to "fix" it for a while creating a `CustomAreHighlight` that is identical to the one you wrote, I've changed only the class name, and loading that instead of the original one -- and again for when it breaks again. I'm basically switching between identical components components, but after a while it breaks again. It happens on all the browsers I've tried it.

![issue](https://media.giphy.com/media/Y48aicbfdXIwTk8JD3/giphy.gif)

Thank you!

**Comment by JAlvezzz** [comment]
Hi guys,

I have used this library because I need the functionality to select a placeholder in pdf files. I found the same issue because in my html page I have some input fields and make necessary to scroll to the component to edit the pdf.

The problem is in **MouseSelection.js** in the following code:

```
const containerCoords = (pageX: number, pageY: number) => {
      if (! containerBoundingRect) {
        containerBoundingRect = container.getBoundingClientRect();
      }

      return {
        x: pageX - containerBoundingRect.left + container.scrollLeft,
        y: pageY - containerBoundingRect.top + container.scrollTop
      };
    };
```

**By changing to:**

```
const containerCoords = (pageX: number, pageY: number) => {
      if (! containerBoundingRect) {
        containerBoundingRect = container.getBoundingClientRect();
        realContainerBoundingRect.left = containerBoundingRect.left;
        realContainerBoundingRect.top = containerBoundingRect.top;

        if (window.scrollY > 0) {
          realContainerBoundingRect.top += window.scrollY;
        }
      }

      return {
        x: pageX - realContainerBoundingRect.left + container.scrollLeft,
        y: pageY - realContainerBoundingRect.top + container.scrollTop
      };
    };

```
The solution for this issue is to get the current value of the window scroll and apply this value to the **container.getBoundingClientRect()** (which applies to the pdf-highlighter component).

**Comment by jinman** [comment]
We are also facing the same issue. Is this possible to include in the build so we can help test it in release? @agentcooper

**Comment by agentcooper** [comment]
Could you please open a pull request with this change?

**Comment by jinman** [comment]
Created a PR. thanks @JAlvezzz

**Comment by agentcooper** [comment]
Fixed in #100.
